.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -57,7 +57,7 @@ env:
   cilium_cli_ci_version: 8c624e141844a19a6b2e21d5255f3ca6195c996d
   check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
   upgrade_cilium_from_vsn: 1.13.4
-  upgrade_cilium_to_vsn: 1.14.0-rc.1 # empty means upgrade to CI build
+  upgrade_cilium_to_vsn: 1.14.0 # empty means upgrade to CI build
```

Comment:
Shouldn't this be always empty anyway? We want to cover the CI builds like we did in K8sUpdates. Otherwise, we'll always learn about regressions after merging.

Reply:
Done. Once this PR is merged, we can try to backport it to the v1.1{1,2,3,4} branches.